<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ihm_csvDataLogger</class>
 <widget class="QMainWindow" name="ihm_csvDataLogger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="6">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="2">
       <widget class="QPushButton" name="PB_refresh_liste">
        <property name="toolTip">
         <string>Mise à jour de la liste des variables disponibles</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../icons.qrc">
          <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pb_decoche_tout">
        <property name="toolTip">
         <string>Décoche tout</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../icons.qrc">
          <normaloff>:/icons/cancel.png</normaloff>:/icons/cancel.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pb_save_variables_liste">
        <property name="toolTip">
         <string>Savegarde la liste des variables observées</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../icons.qrc">
          <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pb_open_variables_liste">
        <property name="toolTip">
         <string>Charge une liste de variables à observer</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../icons.qrc">
          <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pb_affiche_cochees">
        <property name="toolTip">
         <string>Affiche uniquement les données sélectionnées</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../icons.qrc">
          <normaloff>:/icons/select_all.png</normaloff>:/icons/select_all.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="6">
     <widget class="QLineEdit" name="filtre_noms_variables">
      <property name="placeholderText">
       <string>Filtre sur le nom</string>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <widget class="QSpinBox" name="log_period">
      <property name="toolTip">
       <string>Log period</string>
      </property>
      <property name="suffix">
       <string> [msec]</string>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>10000000</number>
      </property>
      <property name="singleStep">
       <number>100</number>
      </property>
      <property name="value">
       <number>500</number>
      </property>
     </widget>
    </item>
    <item row="8" column="6">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="6" colspan="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" rowspan="2">
       <widget class="QPushButton" name="pb_start">
        <property name="toolTip">
         <string>Start logging</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../icons.qrc">
          <normaloff>:/icons/player_play.png</normaloff>:/icons/player_play.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pb_stop">
        <property name="toolTip">
         <string>Stop logging</string>
        </property>
        <property name="icon">
         <iconset resource="../../icons.qrc">
          <normaloff>:/icons/player_stop.png</normaloff>:/icons/player_stop.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pb_pause">
        <property name="toolTip">
         <string>Pause logging</string>
        </property>
        <property name="icon">
         <iconset resource="../../icons.qrc">
          <normaloff>:/icons/player_pause.png</normaloff>:/icons/player_pause.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pb_log_once_manualy">
        <property name="toolTip">
         <string>Log once manualy</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../icons.qrc">
          <normaloff>:/icons/edit_add.png</normaloff>:/icons/edit_add.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="6">
     <widget class="QCheckBox" name="cb_periodic_log">
      <property name="toolTip">
       <string>Log every ... [msec]</string>
      </property>
      <property name="text">
       <string>Periodic</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QCheckBox" name="cb_log_on_value_changed">
      <property name="toolTip">
       <string>Log on any value changed</string>
      </property>
      <property name="text">
       <string>Value Changed</string>
      </property>
     </widget>
    </item>
    <item row="6" column="6">
     <widget class="QCheckBox" name="cb_log_on_value_updated">
      <property name="toolTip">
       <string>Log on any value updated</string>
      </property>
      <property name="text">
       <string>Value Updated</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" rowspan="7" colspan="6">
     <widget class="QListWidget" name="liste_variables">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <property name="uniformItemSizes">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Nouvel élément</string>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="6" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <item>
       <widget class="QLineEdit" name="output_pathfilename">
        <property name="placeholderText">
         <string>output file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pb_select_file">
        <property name="toolTip">
         <string>Select output file</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
